lua/entities/gmod_wire_expression2/core/custom/constraintcore.lua

```diff
@@ -91,7 +93,8 @@ local countLookup = {
 local function checkEnts(self, ent1, ent2)
 	if not IsValid(ent1) and not ent1:IsWorld() then return self:throw("Invalid entity!", false) end
 	if not IsValid(ent2) and not ent2:IsWorld() then return self:throw("Invalid target entity!", false) end
-	if ent1 == ent2 then return self:throw("Cannot constrain an entity to itself!", false) end
+	-- if ent1 == ent2 then return self:throw("Cannot constrain an entity to itself!", false) end
+	-- Constraints actually can be made to themselves - the corresponding constraint functions will handle this part
```

Comment:
Can just delete this part entirely if there's no issues constraining an ent to itself. I see starfall doesn't check this so I think it should be fine

Reply:
Fixed both issues in latest commit